assets/src/scripts/monitoring-location/components/hydrograph/selectors/cursor.js

```diff
@@ -7,9 +7,12 @@ import {getCurrentMethodID} from 'ml/selectors/time-series-selector';
 
 import {getVisibleGroundWaterLevelPoints} from './discrete-data';
 import {getCurrentVariablePointsByTsId} from './drawing-data';
-import {getMainXScale, getMainYScale} from './scales';
+import {getMainXScale, getMainYScale, getTimeRange} from './scales';
 import {isVisible} from './time-series-data';
 
+const isInTimeRange = function(dateTime, timeRange) {
```

Comment:
Utility function check if a dateTime is within a time range. This is used to filter the list of visible points so that we only have the points visible on the desired graph.

Reply:
That's an nice and straight forward solution.